pytest_mh/utils/firewall.py

```diff
@@ -289,95 +302,77 @@ def _next_priority(self) -> int:
         self._priority -= 1
         return priority
 
-    def add_direct_rule(
+    def add_policy(
         self,
-        chain: str,
-        args: list[Any],
+        name: str,
```

Comment:
This parameter is missing in the description

Reply:
Fixed